src/app/client/src/app/modules/org-management/components/status/status.component.spec.ts

```diff
@@ -106,4 +106,17 @@ describe('StatusComponent', () => {
     component.ngOnDestroy();
     expect(component.unsubscribe$.complete).toHaveBeenCalled();
   });
+  it('should call getBulkUploadStatus method and pass else condition if processId contains only whiteSpaces', () => {
```

Comment:
write positive test case also when you pass proper process id it should call the service method